godtools/AppDelegate.swift

```diff
@@ -27,6 +27,8 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
                 UIApplication.shared.isNetworkActivityIndicatorVisible = false
         }
         
+        print(NSHomeDirectory())
```

Comment:
I actually want to leave this one in there for awhile. It's the quickest and easiest way i've found to be able to determine the disk location for the database and resources folder. It shouldn't hurt anything, but we can leave it in for launch.

I could also wrap it in a debug tag so it only runs on debug builds, what do you think?